actor-typed-tests/src/test/java/jdocs/org/apache/pekko/typed/RouterTest.java

```diff
@@ -240,9 +240,8 @@ public void showGroupRoutingWithConsistentHashing() throws Exception {
     // #consistent-hashing
     // Then messages with equal Message.id reach the same actor
     // so the first message in each probe queue is equal to its second
-    // NB: this test can start failing if you change the actor path (eg the URL scheme) - to fix you
-    // will need
-    // to change id2 value until it starts passing again
+    // NB: this test can start failing if you change the actor path (eg the URL scheme)
```

Comment:
Can we update this comment to add a bit more context, i.e.

NB: this test can start failing if you change the actor path (eg the URL scheme) due to these values being hashed (and depending on the resulting hash its possible to change the distribution of actors in the ring which the test relies on)

Reply:
done